portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/ClientExtension.testcase

```diff
@@ -825,6 +825,32 @@ definition {
 			locator1 = "TextInput#ANY");
 	}
 
+	@description = "created (theme SVG extensions), the theme SVG extensions admin page lists the name and type"
```

Comment:
please put in the description the LPS of the ticket, it is easier to find later

Reply:
I will add